### ISSUE TITLE
Weird vision shenaningans

### ISSUE THREAD

**Issue by Enrico-Swagolo** [LoneGazebo/Community-Patch-DLL#3352]
_1. Mod version (i.e Date - 4/23):_

current beat
_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
Basically my Danish berserker embarked on an hostile Bismarck Dromon because the vision of Dromon's tile suddenly disappeared, and then I could do whatever I wanted with my unit that was on the same spot as the enemy unit, so I just put him on regular ground. Also sometimes stuff I should have vision of (my CS, my vassals) suddenly stops giving me any info until I move an unit there, after which the sight is restored.

**Comment by LoneGazebo** [comment]
@ilteroi this seems up your alley. @Enrico-Swagolo if you can post a save for him that'd be lovely.

G

**Comment by Enrico-Swagolo** [comment]
Ilteroi can't load my saves, it never works.

**Comment by LoneGazebo** [comment]
i might be able to. send them.

**Comment by Enrico-Swagolo** [comment]
[ohridetc.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1151184/ohridetc.zip)

I don't have the save for the zerk, but I have for weird vision all over the map.  Check it, especially Ohrid (my main continent, mid-east). I explained the zerk occurence a bit wrong in hindsight, the picture explains it better - I told him to move to a tile on the same continent through water despite him not seeing what was on an adjacent water tile, and after the movement I saw the invisible tile (Bermuda triangle or something?) and it contained a Trireme. Those spots near Byzantium's cities often turn invisible even when I move an unit there, and my vision in these parts and some other sports I find hard to understand. (Germany, Arabia, Byzantium are my vassals right now)
